synapse/cortex.py

```diff
@@ -29,13 +29,57 @@ def __init__(self, name, msg):
         Exception.__init__(self, '%s invalid: %s' % (name, msg))
         self.param = name
 
+storectors = {
+    'lmdb': synapse.cores.lmdb.LmdbStorage,
+    'sqlite': synapse.cores.sqlite.SqliteStorage,
+    'ram': synapse.cores.ram.RamStorage,
+    'postgres': synapse.cores.postgres.PsqlStorage,
+}
+
 corctors = {
-    'lmdb': synapse.cores.lmdb.Cortex,
-    'sqlite': synapse.cores.sqlite.Cortex,
+    'lmdb': synapse.cores.lmdb.initLmdbCortex,
+    'sqlite': synapse.cores.sqlite.initSqliteCortex,
     'ram': synapse.cores.ram.initRamCortex,
-    'postgres': synapse.cores.postgres.Cortex,
+    'postgres': synapse.cores.postgres.initPsqlCortex,
 }
 
+def openstore(url, storconf=None, **opts):
```

Comment:
openurl() is still the preferred operator for people wanting a cortex. openstore() returns JUST the storage object, which doesn't have any sort of data model / storm runtime loaded.